native/cocos/platform/empty/modules/SystemWindow.cpp

```diff
@@ -46,11 +45,15 @@ uintptr_t SystemWindow::getWindowHandle() const {
     return 0;
 }
 
+uint32_t SystemWindow::getWindowId() const {
+    return 0;
+}
```

Comment:
0 is invalid id, is it expected value?

Reply:
I didn't get an error if I returns 0, but yes, I think it's better to return ISystemWindow::mainWindowId